templates/agenda/broadcast.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends 'layout-with-agenda-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda' %}
+
+{% block title %}{% trans %}Broadcast agenda{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Broadcast agenda{% endtrans %}</h1>
+            <p class="text-muted"><small>{% trans %}Send notice to board members informing them that they now has access to the agenda.{% endtrans %}</small></p>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    <main role="main" class="col-12 mt-3">
+        {% if 3 == agenda.status %}
```

Comment:
`3` should probably be replaced by a constant or maybe better a method on agenda named `isReady` or something similar.

Reply:
Added `isReady` function to `Agenda` class.